jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -117,11 +117,12 @@ static const OptimizationStrategy JBwarmStrategyOpts[] =
    { OMR::treeSimplification                                                       },
    { OMR::trivialDeadTreeRemoval,                    OMR::IfEnabled                },
 
-   { OMR::inductionVariableAnalysis,                 OMR::IfLoops                  },
-   { OMR::loopCanonicalization,                      OMR::IfLoops                  },
+   { OMR::basicBlockOrdering                                                       }, // clean up block order going into canonicalizer
```

Comment:
@0xdaryl for methods without loops, not much will happen between this run of block ordering and the subsequent block extension pass (which also does block ordering). So your first suggestion was right, I think. I updated it to only run block ordering OMR::IfLoops.